code/modules/asset_cache/assets/rtd.dm

```diff
@@ -0,0 +1,117 @@
+/datum/asset/spritesheet/rtd
+	name = "rtd-tgui"
+
+/datum/asset/spritesheet/rtd/create_spritesheets()
+	var/list/tiles = list(
```

Comment:
This list is just for convenience.  I copied all floor types from the static list in RTD and slapped it in here.
It's not possible to share this list cause inside the RTD all these types are arranged into categories and subcategories while in here its just one big linear list.
While it is duplicate on a way its not too much work for anyone to just paste their values at the end of this list

Reply:
We don't need to share the list but we should absolutely be creating this dynamically

> Can we instead create an RTD and read its values and extract it from there instead? 

As in, we should be creating a list that loops over those values and flattens it out. Potentially using a GLOB instead of creating the RTD. This is definitely problematic since it doesn't even appear to be linted against, meaning it's very easy to add one in one place and not the other.

Reply:
I could move the static list inside RTD to global_vars and flatten that list here then.  Sure I'll do that